packages/account/src/utils/Constants.ts

```diff
@@ -0,0 +1,8 @@
+import { ethers } from 'ethers'
+import { ChainId } from '@biconomy/core-types'
+
+export const DEFAULT_VERIFICATION_GAS_LIMIT = ethers.BigNumber.from(70000);
+export const DEFAULT_CALL_GAS_LIMIT = ethers.BigNumber.from(35000);
```

Comment:
how is this number picked?

Reply:
took from userOp lib

Reply:
where is this directly used?

Reply:
```
let userOp: UserOperation = { ...DEFAULT_USER_OP }
userOp.sender = await this.getSmartAccountAddress()
userOp.nonce = await this.nonce()
userOp.initCode = userOp.nonce.eq(0) ? await this.getInitCode() : '0x'
userOp.callData = callData

userOp = await this.estimateUserOpGas(userOp)
userOp = await this.getPaymasterAndData(userOp)
return userOp
```
using in in buildOp function to initialise userOp with default values even though later on it will get overrides with actual values